package/android/src/main/java/com/mrousavy/camera/CameraView.kt

```diff
@@ -173,13 +191,15 @@ class CameraView(context: Context) : FrameLayout(context) {
       }
       val cameraId = cameraId ?: throw NoCameraDeviceError()
 
+      val previewSize = previewSize ?: return
```

Comment:
This should probably be `previewView.size`, right? Single source of truth - if we attach a PreviewView we need exactly this size - otherwise it could go out of sync.